src/test/regress/sql/grant_on_database_propagation_from_non_maindb.sql

```diff
@@ -0,0 +1,251 @@
+-- Public role has connect,temp,temporary privileges on database
+-- To test these scenarios, we need to revoke these privileges from public role
+-- since public role privileges are inherited by new roles/users
+set citus.enable_create_database_propagation to on;
+create database test_2pc_db;
+
+show citus.main_db;
+
+revoke connect,temp,temporary  on database test_2pc_db from public;
+
```

Comment:
let's delete trailing newlines in the file

Reply:
Removed